lib/pages/main/components/navigation_drawer_widget.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:mosquito_alert_app/pages/my_reports_pages/my_reports_page.dart';
+import 'package:mosquito_alert_app/pages/settings_pages/campaign_tutorial_page.dart';
+import 'package:mosquito_alert_app/pages/settings_pages/settings_page.dart';
+import 'package:mosquito_alert_app/utils/MyLocalizations.dart';
+import 'package:mosquito_alert_app/utils/UserManager.dart';
+import 'package:mosquito_alert_app/utils/Utils.dart';
+import 'package:mosquito_alert_app/utils/style.dart';
+import 'package:package_info/package_info.dart';
+
+class CustomDrawer extends StatefulWidget {
+  const CustomDrawer({Key? key}) : super(key: key);
+
+  @override
+  State<CustomDrawer> createState() => _CustomDrawerState();
+}
+
+class _CustomDrawerState extends State<CustomDrawer> {
+  var packageInfo;
+
+  @override
+  void initState() {
+    super.initState();
+    getPackageInfo();
+  }
+
+  void getPackageInfo() async {
+    var _packageInfo = await PackageInfo.fromPlatform();
+    setState(() {
+      packageInfo = _packageInfo;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      child: ListView(
+        padding: EdgeInsets.zero,
+        children: [
+          DrawerHeader(
+            child: Row(
+              children: <Widget>[
+                // User score
```

Comment:
Click on the points_box should have the same behaviour as current version and go to:
```
          Navigator.push(
            context,
            MaterialPageRoute(
              builder: (context) => InfoPage(
                "${MyLocalizations.of(context, 'url_point_1')}$userUuid",
              ),
            )
 ```